easycord/composer.py

```
@@ -2,12 +2,14 @@
 from __future__ import annotations
 
 import logging
+from typing import Callable
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)